bundles/framework/mapfull/instance.js

```diff
@@ -177,6 +177,15 @@ Oskari.clazz.define('Oskari.mapframework.bundle.mapfull.MapFullBundleInstance',
                     });
                 }
 
+                let bingMapsPlugin = plugins.find(cur => cur.id === 'Oskari.mapframework.mapmodule.BingMapsLayerPlugin');
```

Comment:
Lets refactor this to some const automagicPlugins = ['Oskari.mapframework.mapmodule.VectorTileLayerPlugin', 'Oskari.mapframework.mapmodule.BingMapsLayerPlugin'] and loop over that for the automatic addition.

Reply:
Moved automagic plugins to a separate file.